packages/mosaic/tags/tag-input.ts

```diff
@@ -212,6 +245,19 @@ export class McTagInput implements McTagTextControl, OnChanges {
         this.inputElement.focus();
     }
 
+    private separatorKeyToSymbol(k): string {
+        const sep = {
+            [ENTER]: '\r\n',
```

Comment:
```suggestion
            [ENTER]: /\r?\n/,
```
Для работы в юниксе (и маке, соответственно) здесь нужна регулярка.

Reply:
ок